.github/workflows/build-nightly.yml

```diff
@@ -7,39 +7,16 @@ on:
   schedule:
     - cron: '0 0 * * *'
   workflow_dispatch:
+
+
 jobs:
-  build-nightly:
-    runs-on: ubuntu-22.04
-    permissions:
-      id-token: write
-      contents: read
-    steps:
-      - uses: aws-actions/configure-aws-credentials@v2
-        with:
-          role-to-assume: ${{ secrets.AWS_WEBIDENTITY_FOR_GITHUB_ACTIONS }}
-          aws-region: us-east-1
-      - uses: actions/checkout@v3
-      - run: | 
-             pwd
-             sudo apt-get install python3-pip
-             pip3 --version
-             sudo pip3 install virtualenv
-             virtualenv venv
-             source venv/bin/activate
-             pip install -e .
-             make -B build
-             deactivate
-             ls dist/
-             aws s3 cp dist/*nightly*.whl s3://nm-github-actions/${{ github.event.repository.name }}/
-             todaytime=`date +%Y%m%d`
-             date '+%Y%m%d-%k:%M:%S' | tee log_${GITHUB_REF_NAME}_nightly_${todaytime}_${GITHUB_SHA:0:7}
-             aws s3 cp log_${GITHUB_REF_NAME}_nightly_${todaytime}_${GITHUB_SHA:0:7} s3://nm-github-actions/${{ github.event.repository.name }}/
-             oldDate=`date --date='-2 month' +%Y%m%d`
-             oldWhl=`(aws s3 ls s3://nm-github-actions/${{ github.event.repository.name }}/ | grep nightly | grep "${oldDate}") || echo "notfound"`
-             if [[ "${oldWhl}" != 'notfound' ]]; then
-                 for oldwhl in $(echo "${oldWhl}" | awk '{print $4}')
-                 do
-                     echo "Remove old build ${oldwhl}"
-                     aws s3 rm s3://nm-github-actions/${{ github.event.repository.name }}/${oldwhl}
-                 done
-             fi
+
+    BUILD-SPARSEZOO_NIGHTLY:
+
+      uses: ./.github/workflows/util.yml
+      with:
+        runs_on: ubuntu-22.04
+        run_id: ${{ github.run_id }}
+        build_type: nightly
+        testmo_project_id: 9
```

Comment:
Shouldn’t this be `2` for sparsezoo? It seems we don’t have a Testmo project with an ID of 9, at least that I can see on my account.

Reply:
I created a new project Build which from testmo it looks its project id is 9.